.rubocop.yml

```diff
@@ -24,9 +24,14 @@ Metrics/LineLength:
   Max: 120
 
 Metrics/MethodLength:
-  Enabled: true
   Max: 20
 
+Metrics/MethodLength:
+  Max: 120
+
+Metrics/ModuleLength:
+  Max: 120
```

Comment:
I think the defaults for these are actually good. It encourages better abstraction, breaking things out into separate modules or classes, and making sure that methods and cyclomatic complexity of methods are not too extreme, making code more readable.